src/commons/utilities.tsx

```diff
@@ -217,3 +217,22 @@ export const getFileExtension = (file: FileType): string => {
 	// @ts-ignore
 	return FileExtensionRegex.exec(file.filename)[1];
 };
+
+export const convertToDecimal = (source: string): string => {
+	let result = '';
+	// eslint-disable-next-line no-plusplus
+	for (let i = 0; i < source.length; i++) {
```

Comment:
just a suggestion to avoid the disable-next-line: I would spread source in an array and do a forEach

Reply:
yes using a foreach will be better